README.md

```diff
@@ -1,10 +1,16 @@
 ## Leaderboard Project
 
+- Project basically allows you to create an API endpoint called with a post request an returns the corresponding gameID with `1-createGameWithPostRequest.js` node.js.
+
+- Then you can fill the corresponding game with either `2-fillGamePostRequest.js` node.js or you can update the score from user interface.
+
+- Refresh button allows you to send a get request to the API that you created.
+
 ## SETUP PROJECT
 
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/201659165-aedd2272-d602-40e8-9510-4f09629be1ad.PNG)

- [ ] You have not followed the [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) 😨 **Gitflow** means that you will create a **development** branch from your **main** branch and then you will make that **development** branch your default branch. After that you will make **feature** branches from that **development** branch and after getting approvals on your **feature** branches you will merge them in your **development** branch. You are not going to use **main** branch anywhere for now 👍